src/subscription_manager/cli_command/attach.py

```diff
@@ -94,16 +94,13 @@ def _read_pool_ids(self, f):
                 self.options.pool.append(pool)
 
     def _short_description(self):
-        return _(
-            "Attach a specified subscription to the registered system, when system does not use "
-            "Simple Content Access mode"
-        )
+        return _("Deprecated, this command will be removed from the future major releases")
```

Comment:
Thanks for the feedback, I was a little confused why the jira wanted me to replace all the help text with this standard deprecation message instead of just appending it to the existing help text. Thanks for clarifying this point :+1: